CHANGELOG.md

```diff
@@ -8,8 +8,6 @@ Categories: Added, Removed, Changed, Fixed, Nonfunctional, Deprecated
 
 <!--- Example CHANGELOG entry
 
-## 0.1.0 (2019.07.02)
-
```

Comment:
I assume you're planning a separate release PR, then?

Reply:
yes.